File: POO/temas/20/py/20.2.py
encontrado = False
i = 1
while (i <= 900 and not encontrado):
    if i % 7 == 0 and i % 23 == 0 and i % 2 == 0:
        encontrado = True
    else:
        i = i + 1

print(f"encontrado: {encontrado}")
print(f"i: {i}")

File: working/temas/busqueda/rb/20.2.rb
encontrado = false
i = 1
while i <= 900 && !encontrado
    if i % 7 == 0 && i % 23 == 0 && i % 2 == 0
        encontrado = true
    else
        i = i + 1
    end
end

puts "encontrado: #{encontrado}"
puts "i: #{i}"
